docs/CONTRIBUTING.md

```diff
@@ -228,6 +228,52 @@ Add "WIP" (Work in Progress) or "Draft" to the title if the PR is not yet ready
    ```console
    git branch -d <branch-name>
    ```
+   
+## Choosing the correct branch when contributing
+
+We are using Netlify to publish our pages.
+We have staging and production levels for our documentation which are as follows:
+
+1. Development documentation aka staging (build of `main` branch) - [link](https://main.lifecycle.keptn.sh)
+2. Official documentation aka production (build of `page` branch) - [link](https://lifecycle.keptn.sh)
+
+Let's take a look what happens when your changes are merged in `main` and `page` branch respectively.
+
+### Development documentation (Staging)
+
+This page reflects the current development status of the documentation.
+It will be built regularly and can be easily accessed.
+
+- build: on each push to `main` with documentation changes
+- build-environment: main
+- config folder: [main](./config/staging/)
+
+It should allow bleeding-edge users and contributors to see the current state and help with debugging etc.
+
+### Official documentation (Production)
+
+This documentation set contains all released versions of KLT and is stored in an orphaned branch called `page`.
+
+- build: on each push to `page` with documentation changes
+- build-environment: production
+- config folder: [production](./config/production/)
+
+## FAQs
+
+- Who/how do these changes get reviewed/approved/merged?
+```
+Like every normal PR - if the changes are approved by the maintainer, it will get merged.
+```
+
+- Which branch should I consider when contributing?
+```
+New writing goes to the main branch and can be viewed on the Releases -> development dropdown menu
+```
+
+- When we push the main branch to production, does it overwrite changes that were only pushed to page?
+```
+No, because a new version is generated - hence old page branch will be copied over a “version” folder
+```
```

Comment:
I would put the questions into the paragraphs that talk about the related sections instead of doing an FAQ here.
- how do changes get reviewed/approved: i don't think this question is really necessary here, it's explained in the PR creation paragraphs already.
- which branch should i consider: this could be added to "choosing the correct branch when contributing"
- are changes overwritten: also for this, i would add a sentence to the "choosing the correct branch when contributing" section

Reply:
Done :+1: